corehq/apps/reports/standard/export.py

```diff
@@ -401,10 +399,7 @@ def template_context(self):
     @property
     @memoized
     def saved_exports(self):
-        exports = [
-            self.export_schema.wrap(doc.to_json())
-            for doc in filter(lambda x: x.type == self.export_type, stale_get_exports(self.domain))
-        ]
```

Comment:
This was basically the functionality I mimicked in `HQExportSchema.get_stale_exports`.